react/features/calendar-sync/middleware.js

```diff
@@ -129,6 +129,11 @@ function _fetchCalendarEntries(
         maybePromptForPermission,
         forcePermission
 ) {
+    if (!isCalendarEnabled()) {
+        // Calendar access is not enabled by the SDK.
+        return;
+    }
+
```

Comment:
I'd really have liked more! Something along the lines of disabling the whole feature if the app hasn't enabled the calendar support. Because now the Calendar tab is on the WelcomePage even when the app hasn't enabled the calendar support which feels <negative adjective here> than I'd have liked.